azure-keyvault/azure/keyvault/models/administrator_details.py

```diff
@@ -32,9 +32,9 @@ class AdministratorDetails(Model):
         'phone': {'key': 'phone', 'type': 'str'},
     }
 
-    def __init__(self, first_name=None, last_name=None, email_address=None, phone=None):
-        super(AdministratorDetails, self).__init__()
-        self.first_name = first_name
-        self.last_name = last_name
-        self.email_address = email_address
-        self.phone = phone
+    def __init__(self, **kwargs):
```

Comment:
@lmazuel it looks like the python autorest generation has changed to utilize kwargs for 2.7 and keyword only arguments for 3.4+.  This is a backwards breaking change and it fails our tests which specify arguments by position.  Is this an intentional breaking change or do I need to specify additional arguments to autorest to continue to have it generate positional arguments?

Reply:
This is intentional breaking change in Autorest.Python 3.0. Every major version should take this opportunity to move to this new version, but I still release minor update in Autorest.Python 2.1.40 if necessary.
Method signature in models was a nightmare to keep backward incompatibilities, and created undetected breaking changes several times (the only package I manually edited the code to make it compatible again was this exact issue...). This is simply because compatibility in JSON RestAPI is most flexible that a positional method signature (e.g. order in JSON doesn't matter, but matter in positional arguments).